dali/pipeline/operators/geometric/bb_flip.cc

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/pipeline/operators/geometric/bb_flip.h"
+
+namespace dali {
+
+DALI_REGISTER_OPERATOR(BbFlip, BbFlip, CPU);
+
+
+DALI_SCHEMA(BbFlip)
+                .DocStr(R"code(Operator for horizontal flip (mirror) of bounding box.
+                               Input: Bounding box coordinates; in either (x1,y1,w,h)
+                               or (x1,y1,x2,y2) format)code")
+                .NumInput(1)
+                .NumOutput(1)
+                .AddArg("coordinates_type",
```

Comment:
Please refer to the https://github.com/NVIDIA/DALI/blob/master/dali/pipeline/operators/displacement/flip.cc.
I think we should provide similar API - I mean to give ability to flip horizontally or vertically.
Also this file could be moved to displacement folder.

Reply:
In #110 there is:
```
  .AddOptionalArg("ltrb",
      R"code(If true, bboxes are returned as [left, top, right, bottom], else [x, y, width, height].)code",
      false)
```
I think the same name could be used for consistency.

Reply:
Added vertical flip, edited API for consistency